docs/inference/examples/bbh.rst

```diff
@@ -0,0 +1,207 @@
+.. _inference_example_bbh:
+
+---------------------
+Simulated BBH example
+---------------------
+
+This example recovers the parameters of a simulated binary black-hole (BBH)
+that has similar parameters has GW150914.
+
+^^^^^^^^^^^^^^^^^^^^^^^
+1. Create the injection
+^^^^^^^^^^^^^^^^^^^^^^^
+
+First, we need to create an ``injection.hdf`` file that specifies the
+parameters of the simulated signal. To do that we will use
+``pycbc_create_injection``. Like ``pycbc_inference``,
+``pycbc_create_injections`` uses a configuration file to set the parameters of
+the injections it will create. To create a binary-black hole with parameters
+similar to GW150914, use the following configuration file:
+
+.. literalinclude:: ../../../examples/inference/bbh-injection/injection.ini
+   :language: ini
+
+:download:`Download <../../../examples/inference/bbh-injection/injection.ini>`
+
+Note the similarity to the configuration file for ``pycbc_inference``: you must
+have a ``[variable_params]`` section. If we wanted to randomize one or more
+of the parameters, we would list them there, then add ``[prior]`` sections to
+specify what distribution to draw the parameters from. In this case, however,
+we want to fix the parameters, so we just put all of the necessary parameters
+in the ``[static_params]`` section.
+
+To create the injection file, run:
+
+.. literalinclude:: ../../../examples/inference/bbh-injection/make_injection.sh
+   :language: bash
+
+:download:`Download <../../../examples/inference/bbh-injection/make_injection.sh>`
+
+This will create the ``injection.hdf`` file, which we will give to
+``pycbc_inference``. For more information on generating injection files, run
+``pycbc_create_injections --help``.
+
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2. Setup the configuration files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now we need to set up the configuration for ``pycbc_inference``. Since we
+will be analyzing data, we will need to provide several additional options in a
+``[data]`` section. To keep the configuration files easy to read, we will split
+the data, sampler, and prior settings into their own configuration files.
+
+Here are the model and prior settings we will use:
+
+.. literalinclude:: ../../../examples/inference/priors/gw150914_like.ini
+   :language: ini
+
+:download:`Download <../../../examples/inference/priors/gw150914_like.ini>`
+
+In the ``[model]`` section we have set the model to be ``gaussian_noise``.  As
+described above, this is the standard model to use for CBC signals. It assumes
+that the noise is wide-sense stationary Gaussian noise. Notice that we had to
+provide additional arguments for the low frequency cutoff to use in each
+detector. These values are the lower cutoffs used for the likelihood integral.
+(See the |GaussianNoise| docs for details.)
```

Comment:
The shortcuts for the `GaussianNoise` and `MarginalizedPhaseGaussianNoise` models are broken now. Copy the lines at the top of inference.rst and put them at the top of this file to fix that.